dashboard/app/models/user.rb

```diff
@@ -76,7 +76,7 @@
 
 class User < ActiveRecord::Base
   include SerializedProperties
-  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender)
+  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender races race_interstitial_shown)
```

Comment:
"Prefer not to say" is tricky because technically that can be selected along with other races (maybe that should not be possible, even if it brings up some UI questions).

I've been thinking about this and your other comment about prerendering, though - maybe rather than setting the "shown" flag based on whether the dialog has been rendered by the server, we should flag based on whether the user has interacted with the dialog. That means if they leave the page without touching anything, it will still show up again, but that's probably fine, and it would simplify the data side of things (basically, if they hit "Close" we'd mark that as a "Prefer not to say" opt-out).

I think I'm going to rework it and set it up that way, once I verify that it's ok to treat hitting "close" as equivalent to "Prefer not to say".